modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -743,6 +743,43 @@ class CV_EXPORTS_W QRCodeDetector
             OutputArrayOfArrays straight_qrcode = noArray()
     ) const;
 
+
+#ifndef CV_DOXYGEN  // COMPATIBILITY
```

Comment:
In most cases we use doxygen directives to ignore block of code:
```.cpp
//! @cond IGNORED 
...
//! @endcond
```

Reply:
Lets keep it as is - there is no benefits in that replacement.